city_scrapers/spiders/il_capital_development.py

```diff
@@ -0,0 +1,91 @@
+from datetime import datetime, timedelta
+
+from city_scrapers_core.constants import BOARD
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class IlCapitalDevelopmentSpider(CityScrapersSpider):
+    name = "il_capital_development"
+    agency = "Illinois Capital Development Board"
+    timezone = "America/Chicago"
+    start_urls = ["https://www2.illinois.gov/cdb/about/boardmeetings"]
+    location = {
+        "name": "James R. Thompson Center",
+        "address": "100 West Randolph Street, 14th Floor, Chicago, IL 60601"
+    }
+
+    def parse(self, response):
+        """
```

Comment:
Small thing, but it's probably fine to remove this comment if we aren't using it